src/internal/connector/graph_connector_onedrive_test.go

```diff
@@ -110,6 +120,26 @@ func (suite *GraphConnectorOneDriveIntegrationSuite) SetupSuite() {
 	suite.secondaryUser = tester.SecondaryM365UserID(suite.T())
 	suite.acct = tester.NewM365Account(suite.T())
 
+	// Not using m365.Users to avoid cyclic imports
+	users, err := discovery.Users(ctx, suite.connector.Owners.Users(), fault.New(true))
+	require.NoError(suite.T(), err, "fetching users")
+
+	for _, u := range users {
+		if *u.GetUserPrincipalName() == suite.user {
+			suite.userID = *u.GetId()
+		} else if *u.GetUserPrincipalName() == suite.secondaryUser {
+			suite.secondaryUserID = *u.GetId()
+		}
+	}
```

Comment:
I think you can just use `suite.connector.Owners.Users().GetByID(ctx, <user name>)` to get a single `models.Userable` graph SDK struct for a given user ID or principal name (PN). PNs look a lot like emails and I think all the users we have right now have the same email and PN